modules/distr-tracing-tempo-install-tempomonolithic-cli.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-install-tempomonolithic-cli_{context}"]
+= Installing a TempoMonolithic instance by using the CLI
+
+:FeatureName: The TempoMonolithic instance
+include::snippets/technology-preview.adoc[]
+
+You can install a TempoMonolithic instance from the command line.
```

Comment:
```suggestion
You can install a `TempoMonolithic` instance from the command line.
```

Reply:
TY. As explained in reply to another comment.